functions-and-operators/json-functions.md

```diff
@@ -6,80 +6,156 @@ aliases: ['/docs/dev/functions-and-operators/json-functions/','/docs/dev/referen
-| [JSON_OBJECTAGG(key, value)](https://dev.mysql.com/doc/refman/8.0/en/aggregate-functions.html#function_json-objectagg) | Provides an aggregation of values for a given key. |
+| [JSON_ARRAYAGG()](/functions-and-operators/json-functions/json-functions-aggregate.md#json_arrayagg) | Provides an aggregation of keys. |
+| [JSON_OBJECTAGG()](/functions-and-operators/json-functions/json-functions-aggregate.md#json_objectagg) | Provides an aggregation of values for a given key. |
+
+## Validation functions
+    
+| Function Name                     | Description |
+| --------------------------------- | ----------- |
+| [JSON_SCHEMA_VALID()](/functions-and-operators/json-functions/json-functions-validate.md#json_schema_valid) | Validate a JSON document against a schema to ensure data integrity and consistency. |
+
+## JSON Path
+
+Many of JSON functions use JSON Path to select parts of a JSON document.
+
+| Symbol         | Description                  |
+| -------------- | ---------------------------- |
+| `$`            | Document Root                |
+| `.`            | Member selection             |
+| `[]`           | Array selection              |
+| `*`            | Wildcard                     |
+| `**`           | Path wildcard                |
+| `[<n> to <n>]` | Array range selection        |
+
+Examples:
+
+The JSON to demonstrate this:
+
+```json
+{
+    "database": {
+        "name": "TiDB",
+        "features": [
+            "distributed",
+            "scalable",
+            "relational",
+            "cloud native"
+        ],
+        "license": "Apache-2.0 license",
+        "versions": [
+            {
+                "version": "v8.1.0",
+                "type": "lts",
+                "release_date": "2024-05-24" 
+            },
+            {
+                "version": "v8.0.0",        
+                "type": "dmr",
+                "release_date": "2024-03-29"
+            }
+        ]
+    },
+    "migration_tool": {
+        "name": "TiDB Data Migration",
+        "features": [
+            "MySQL compatible",            
+            "Shard merging"
+        ],
+        "license": "Apache-2.0 license"
+    }
+}
+```
+
+| JSONPath                              | Description                             | Example with [`JSON_EXTRACT()`](/functions-and-operators/json-functions/json-functions-search.md#json_extract) | 
+|-------------------------------------- |-----------------------------------------|-------------------------------|
+| `$`                                   | The root of the document                |                               |
+| `$.database`                          | The database attribute                  |                               |
+| `$.database.name`                     | The name of the database.               | `"TiDB"`                      |
+| `$.database.features`                 | All database features                   |                               |
```

Comment:
I think we could show the full array here in the example?
`["distributed", "scalable", "relational", "cloud native"] `

Reply:
I'll add that.